src/wp-includes/class-wp-image-editor-gd.php

```diff
@@ -323,7 +323,13 @@ public function crop( $src_x, $src_y, $src_w, $src_h, $dst_w = null, $dst_h = nu
 			$dst_h = $src_h;
 		}
 
-		$dst = wp_imagecreatetruecolor( $dst_w, $dst_h );
+		foreach( array( $src_x, $src_y, $src_w, $src_h, $dst_w, $dst_h ) as $value ) {
```

Comment:
`$dst_w` and `$dst_h` can be null. will this need to first make sure they're not passed in as null before  checking if they're numeric/greater-than?

Reply:
Yes, but the lines before this `foreach` check if `dst_w` and/or `dst_h` are falsey. If yes, then they are overwritten and set to the value of `$src_w` and `$src_h` respectively.

Reply:
@iandunn What do you think?

Reply:
doh, i missed that, thanks!

Reply:
No worries. I did a few dohs in this PR myself. 😉